update-dois/src/main/java/org/reactome/release/updateDOIs/UpdateDOIs.java

```diff
@@ -67,17 +75,17 @@ public static void findAndUpdateDOIs(long authorIdTR, long authorIdGK, String pa
 						String dbId = trDOI.getAttributeValue("DB_ID").toString();
 
 						// Used to verify that report contents are as expected, based on provided list from curators
-						fetchHits++;
 						if (expectedUpdatedDOIs.get(updatedDoi) != null && expectedUpdatedDOIs.get(updatedDoi).get("displayName").equals(nameFromDb))
-						{	
+						{
 							updated.add(updatedDoi);
 						} else {
 							String doiWithName = updatedDoi + ":" + nameFromDb;
 							notUpdated.add(doiWithName);
-							continue;
+							if (!testMode) {
+								continue;
+							}
 						}
 						// This updates the 'modified' field for Pathways instances, keeping track of when changes happened for each instance
-						// TODO: Put this after the update to GK Central so that we make sure values match
 						trDOI.getAttributeValuesList("modified");
```

Comment:
Not urgent, but most of the attribute names that exist in `ReactomeJavaConstants` should probably be replaced with the appropriate constant.